cl/scrapers/management/commands/clone_from_cl.py

```diff
@@ -350,6 +352,7 @@ def clone_docket(
     docket_ids: list,
     add_docket_entries: bool,
     add_audio_files: bool,
+    add_clusters: bool,
```

Comment:
Everything looks, you could just add the missing parameters to the docstring: add_audio_files and add_clusters

Reply:
Thanks @quevon24  I have updated the docstrings and set for automerge